subprojects/logging/src/main/java/org/gradle/internal/logging/LoggingManagerInternal.java

```diff
@@ -50,4 +50,9 @@
     LoggingManagerInternal captureStandardError(LogLevel level);
 
     LoggingManagerInternal setLevelInternal(LogLevel logLevel);
+
+    /**
+     * Sets the worker count to display in the UI.
+     */
+    LoggingManagerInternal setMaxWorkerCount(int maxWorkerCount);
```

Comment:
I agree. At this point, we aren't setting a maximum worker count but rather asserting a worker count. What we do with that worker count depends on the UI behavior. I was more leaning toward `setWorkerCount` and dropping the _Max_ prefix. What do you think?